src/toil_scripts/tools/preprocessing.py

```diff
@@ -233,33 +388,44 @@ def run_base_recalibration(job, cores, indel_bam, indel_bai, ref, ref_dict, fai,
     :param str fai: Reference index FileStoreID
     :param str dbsnp: DBSNP VCF FileStoreID
     :param str mem: Memory value to be passed to children. Needed for CI tests
+    :param str bqsr: BQSR recalibration table FileStoreID
     :param bool unsafe: If True, runs gatk UNSAFE mode: "-U ALLOW_SEQ_DICT_INCOMPATIBILITY"
     :return: FileStoreID for the processed bam
     :rtype: str
     """
     work_dir = job.fileStore.getLocalTempDir()
-    file_ids = [ref, fai, ref_dict, indel_bam, indel_bai, dbsnp]
-    inputs = ['ref.fasta', 'ref.fasta.fai', 'ref.dict', 'sample.indel.bam', 'sample.indel.bai', 'dbsnp.vcf']
+    file_ids = [ref, fai, ref_dict, indel_bam, indel_bai, dbsnp, mills]
+    inputs = ['ref.fasta', 'ref.fasta.fai', 'ref.dict', 'sample.bam', 'sample.bai',
+              'dbsnp.vcf', 'mills.vcf']
     for file_store_id, name in zip(file_ids, inputs):
         job.fileStore.readGlobalFile(file_store_id, os.path.join(work_dir, name))
     # Call: GATK -- IndelRealigner
     parameters = ['-T', 'BaseRecalibrator',
                   '-nct', str(cores),
                   '-R', '/data/ref.fasta',
-                  '-I', '/data/sample.indel.bam',
+                  '-I', '/data/sample.bam',
                   '-knownSites', '/data/dbsnp.vcf',
-                  '-o', '/data/sample.recal.table']
+                  '-knownSites', '/data/mills.vcf',
+                  '-o', '/data/recal_data.table']
+    if bqsr:
+        # Best practices recommends doing two passes
+        # https://software.broadinstitute.org/gatk/documentation/article?id=2801
```

Comment:
Do we support this in the pipeline? I don't think we do—and honestly, I think that recommending two pass recalibration has no basis in fact—so I would strip this out.